hugo/content/installation-plus.md

```diff
@@ -0,0 +1,387 @@
+---
+title: "Install NGINX Agent From NGINX Plus Repository"
+draft: false
+description: "Learn how to install NGINX Agent from NGINX Plus repository."
+weight: 400
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-1217"
+categories: ["configuration"]
+doctypes: ["task"]
+---
+
+
+## Prerequisites
+- An NGINX Plus subscription (purchased or trial)
+- NGINX Plus installed. Once installed, ensure it is running. If you don't have it installed already, follow these steps to install [NGINX Plus](https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-plus/)
+- A [supported operating system and architecture](../technical-specifications/#supported-distributions)
+- `root` privilege
+- Your credentials to the MyF5 Customer Portal, provided by email from F5, Inc.
+- Your NGINX Plus certificate and public key (`nginx-repo.crt` and `nginx-repo.key` files), provided by email from F5, Inc.
+
+## Configure NGINX Plus Repository for installing NGINX Agent
+
+Before you install NGINX Agent for the first time on your system, you need to set up the `nginx-agent` packages repository. Afterward, you can install and update NGINX Agent from the repository.
+
+- [Installing NGINX Agent on RHEL, CentOS, Rocky Linux, AlmaLinux and Oracle Linux](#installing-nginx-agent-on-rhel-centos-rocky-linux-almalinux-and-oracle-linux)
+- [Installing NGINX Agent on Ubuntu](#installing-nginx-agent-on-ubuntu)
+- [Installing NGINX Agent on Debian](#installing-nginx-agent-on-debian)
+- [Installing NGINX Agent on SLES](#installing-nginx-agent-on-sles)
+- [Installing NGINX Agent on Alpine Linux](#installing-nginx-agent-on-alpine-linux)
+- [Installing NGINX Agent on Amazon Linux](#installing-nginx-agent-on-amazon-linux)
+
+### Installing NGINX Agent on RHEL, CentOS, Rocky Linux, AlmaLinux and Oracle Linux
+
+1. Create the `/etc/ssl/nginx` directory:
+
+    ```shell
+    sudo mkdir -p /etc/ssl/nginx
+    ```
+
+1. Log in to [MyF5 Customer Portal](https://account.f5.com/myf5/) and download your `nginx-repo.crt` and `nginx-repo.key` files.
+
+1. Copy the files to the `/etc/ssl/nginx/` directory:
+
+    ```shell
+    sudo cp nginx-repo.crt nginx-repo.key /etc/ssl/nginx/
+    ```
+
+1. Install the prerequisites:
+
+    ```shell
+    sudo yum install yum-utils procps
+    ```
+
+1. To set up the yum repository, create the file named `/etc/yum.repos.d/nginx-agent.repo` with the following contents:
+
```

Comment:
Need to add command to this step create repo file in that directory. See step 2 from existing doc https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-open-source/#installing-a-prebuilt-rhelcentosoracle-linuxalmalinuxrocky-linux-package-from-the-official-nginx-repository
``` shell
 sudo vi /etc/yum.repos.d/nginx-agent.repo

```